tests/playwright/sanity/image-carousel.test.js

```diff
@@ -7,17 +7,20 @@ test( 'Image Carousel', async ( { page }, testInfo ) => {
 		editor = await wpAdmin.useElementorCleanPost();
 
   // Close Navigator
-  await page.click( '#elementor-navigator__close' );
+  await editor.closeNavigatorIfOpen();
 
-	// Act.
-	await editor.addWidget( 'image-carousel' );
+  // Set Canvas template.
+  await editor.useCanvasTemplate();
 
-  await page.locator( '[aria-label="Add Images"]' ).click();
+  // Act.
+  await editor.addWidget( 'image-carousel' );
 
-  // Open Media Library
-  await page.click( 'text=Media Library' );
+  // Hide slider navigation.
+  await page.selectOption( '.elementor-control-navigation >> select', 'none' );
 
   // Upload the images to WP media library
+  await page.locator( '[aria-label="Add Images"]' ).click();
```

Comment:
can this be extracted to the editor ?
something like  ...
editor.addImages  (ImageArray)

Reply:
@TzviRabinovitch 
We have a basis function for the that inside the `editor-page.js` file: `populateImageCarousel()`. I have used that one now.